zerver/openapi/zulip.yaml

```diff
@@ -3748,3 +3563,99 @@ components:
         items:
           type: string
       required: false
+    Narrow:
+      name: narrow
+      in: query
+      description: |
+        A JSON-encoded array of length 2 indicating the narrow for which you'd
+        like to receive events for. For instance, to receive events for the
+        stream `Denmark`, you would specify `narrow=['stream', 'Denmark']`.
+        Another example is `narrow=['is', 'private']` for private messages.
+        Default is `[]`.
+      schema:
+        type: array
```

Comment:
What I actually meant in the last review was that `default: [ ]` should be present instead `default: narrow = [ ]`. Everything else looks good.